### PR TITLE
Adjust Draft

### PR DESCRIPTION
**Pull request by JKRhb**
[Latest rendered version](https://namib.me/draft-coap-mud/expand-draft2/draft-romann-mud-constrained.html)

See [here](https://hackmd.informatik.uni-bremen.de/X0SsUEsIQxeUGT3b0RiWhQ#) for notes regarding the draft (in German).

### CODE REVIEW

File: draft-jimenez-t2trg-mud-coap.md [hunk 1]
```diff
@@ -1,43 +1,74 @@
...
+  github: namib-project/draft-coap-mud
 
 author:
+ -
+    ins: J. Romann
+    name: Jan Romann
+    organization: Universität Bremen
+    email: jan.romann@uni-bremen.de
+    role: editor
  -
```
Comment (JKRhb) [inline draft-jimenez-t2trg-mud-coap.md:27]:
```suggestion
 -
    ins: H. Damer
    name: Hugo Hakim Damer
    organization: Universität Bremen
    email: hdamer@uni-bremen.de
    role: editor
 - 
```

File: draft-jimenez-t2trg-mud-coap.md [hunk 2]
```diff
@@ -1,43 +1,74 @@
...
     organization: Ericsson
     phone: "+358-442-992-827"
     email: jaime@iki.fi
 
+entity:
+        SELF: "[RFC-XXXX]"
+
 --- abstract
-This document provides some guidelines on how to add Manufacturer Usage Descriptions (MUD) to CoAP environments. We propose the use of URLs using the "coaps://" scheme and the hosting of the MUD files on the endpoints themselves.
+This document provides some guidelines on how to add Manufacturer Usage Descriptions (MUD) to constrained environments.
```
Comment (JKRhb) [inline draft-jimenez-t2trg-mud-coap.md:38]:
```suggestion
This document specifies new methods for adding Manufacturer Usage Descriptions (MUD) to constrained environments.
```

File: draft-jimenez-t2trg-mud-coap.md [hunk 3]
```diff
@@ -162,43 +193,71 @@ MUDstring:              String containing a MUD URL as defined
...
+Resources which either host MUD URLs or MUD files MAY also be indicated using the CoRE Link Format !{{RFC6690}}.
+For this purpose, additional link parameters are defined:
+With the link relation-types `mud-file` and `mud-url`, a link MAY be annotated as pointing to a MUD file or a MUD URL, respectively.
+Note that the use of these relation-types is not limited to constrained environments and can also be used to annotate links in other contexts as well.
+
+<!-- TODO: Mention resource-type and /.well-known/core -->
+
+<!-- TODO: Add example -->
 
 ### Resource Directory
```
Comment (JKRhb) [inline draft-jimenez-t2trg-mud-coap.md:237]: TODO: relation-type vs resource type

File: draft-jimenez-t2trg-mud-coap.md [hunk 4]
```diff
@@ -208,20 +267,20 @@ REQ: GET coap://rd.company.com/rd-lookup/res?rt=mud
...
-     rd-item <coap://[2001:db8:3::102]/mud/switch> { rt "mud" }
-     rd-item <coap://[2001:db8:3::102]/mud/lock> { rt "mud" }
-     rd-item <coap://[2001:db8:3::103]/mud/light> { rt "mud" }
+     rd-item <coap://[2001:db8:3::101]/mud/box> { rel "mud-file" }
+     rd-item <coap://[2001:db8:3::102]/mud/switch> { rel "mud-file" }
+     rd-item <coap://[2001:db8:3::102]/mud/lock> { rel "mud-file" }
+     rd-item <coap://[2001:db8:3::103]/mud/light> { rel "mud-file" }
 ~~~
 
 ### Multicast
```
Comment (JKRhb) [inline draft-jimenez-t2trg-mud-coap.md:276]: Possible additions:
- "All MUD nodes" multicast address
- "All MUD controllers" multicast address (or reuse "All Routers address"?)
    - Alternative: All MUD Receivers

Comment (JKRhb) [reply draft-jimenez-t2trg-mud-coap.md:276]: - Requirement: packets need to be allowed to get through
- Scopes of addresses used for multicast?

Comment (JKRhb) [reply draft-jimenez-t2trg-mud-coap.md:276]: Well-known endpoint for controllers

Comment (JKRhb) [reply draft-jimenez-t2trg-mud-coap.md:276]: Additional methods for finding the interface(s) of MUD controllers.

Comment (JKRhb) [reply draft-jimenez-t2trg-mud-coap.md:276]: Restructure document: Well-known resources first, then methods for discovery.

**Review by pulsastrix — APPROVED** [review]

**Pull request merged**